package.json

```diff
@@ -5,11 +5,11 @@
   "main": "lib/objection.js",
   "license": "MIT",
   "scripts": {
-    "test": "npm run eslint && mocha --slow 10 --timeout 15000 --reporter spec --recursive tests",
```

Comment:
ugly hacks like these are exactly why I prefer to have my spec files colocated with files and names with a suffix `.spec.js`. Then it's just about running all the files with that suffix. Much easier than trying to conform every spec file to a single directory.

Reply:
Sorry, I didn't realize mocha would require that and blow up. Maybe the whole `files` folder should be moved elsewhere, but I'll do that. I'll merge this now.